nginx/nginx_error_pages/generate.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
can you please update the [readme.md](https://github.com/medic/cht-core/blob/dn/8179-nginx-json/nginx/README.md) with how and when to use this script?

Reply:
PR updated, I put the readme in the `nginx_error_pages` directory as the other readme seems more targeted to users of the image.

Reply:
to the point for anyone who goes looking - thank you!